src/sql/magic.py

```diff
@@ -38,6 +38,17 @@
 
 from sql.telemetry import telemetry
 
+from IPython import get_ipython
+
+try:
+    from comm import get_comm_manager
+except ImportError:
+    def get_comm_manager():
```

Comment:
Do we have test case for this function!

Reply:
This was just a test for initializing comm. removed it.